Tools/autotest/sim_vehicle.py

```diff
@@ -1067,56 +1107,84 @@ def generate_frame_help():
 if cmd_opts.frame is None:
     cmd_opts.frame = vinfo.options[cmd_opts.vehicle]["default_frame"]
 
-# setup ports for this instance
-mavlink_port = "tcp:127.0.0.1:" + str(5760 + 10 * cmd_opts.instance)
```

Comment:
Were those unused?

Reply:
I factored this line (and the one after) into start_mavproxy
https://github.com/ArduPilot/ardupilot/pull/13405/files/b2ae81f729e513541d9f1d6bf84e1e0f324a9851#diff-460241a402d32677bcde6618e6e880deR749-R751